source/collimation.f90

```diff
@@ -2498,28 +2453,10 @@ subroutine collimate_do_collimator(stracki)
     end if
   end if
 
-!++  Assign aperture which we define as the FULL width (factor 2)!!!
-!JUNE2005 AGAIN, SOME SPECIFIC STUFF FOR RHIC
-  if(cdb_cNameUC(icoll)(1:4).eq.'COLM') then
-    c_aperture = two*calc_aperture
-    nom_aperture = two*nom_aperture
```

Comment:
ok here we break the user interface to L-shaped collimators.
We have to think about how to re-implement L-shaped collimators...

Reply:
Yes, we have to implement this in a different way.